.env

```diff
@@ -23,6 +23,8 @@ _APP_DB_SCHEMA=appwrite
 _APP_DB_USER=user
 _APP_DB_PASS=password
 _APP_DB_ROOT_PASS=rootsecretpassword
+_APP_PROJECT_DB=db_fra1_02=mysql://user:password@mariadb:3306/appwrite
```

Comment:
Naming conventions requires some more work.

Reply:
Naming convention of the environment variable itself or the database name ?